cloud_dashboard/src/service/utility.ts

```diff
@@ -206,3 +221,69 @@ export const getEntityDataAll = async (entityTypeId: string) => {
   }
   return output;
 };
+
+/**
+ * Getter of LaunchTemplateView's URL for CloudContext.
+ * @param cloudContext CloudContext
+ * @returns URL
+ */
+export const getLaunchTemplateViewUrl = (cloudContext: CloudContext) => {
+  if (cloudContext.name === 'ALL') {
+    return `/${cloudContext.cloudServiceProvider as string}/server_template`;
+  } else {
+    return `/server_template/${cloudContext.cloudServiceProvider as string}/${cloudContext.name}`;
+  }
```

Comment:
@RyoYamashita765 Can we refactor:

```suggestion
  return cloudContext.name === 'ALL'
    ? `/${cloudContext.cloudServiceProvider as string}/server_template`
    : `/server_template/${cloudContext.cloudServiceProvider as string}/${cloudContext.name}`;
```